### PR TITLE
Feature/all organizations view

### LINKED ISSUE #16: Create all organizations view
Create initial markup for all organizations view. Include i18n tags.

Definition of done
---
* [x] Basic markup exists, optionally using CSS framework semantics.
* [x] i18n tags have been included for any UI texts.
* [x] Subscription to all organizations publication exists.

### PR DESCRIPTION
**Pull request by frenchbread**
Closes #16 

# Proposed changes
- Add All Organizations view
- Add `/organizations` link to navigation header

@Digipalvelutehdas/dipor-team please review.

Yet, this PR contains commits that are not a part of this issue and have not been merged to `develop`. Do not merge this until the dependent PRs are merged.

### CODE REVIEW

**Comment by frenchbread** [comment]
Moved `router.js` file into "organizations" folder.

File: docs/en.i18n.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "navigation-home": "Home",
```
Comment (brylie) [inline docs/en.i18n.json]: Perhaps change these prefixes from `navigation-` to `mainLayoutNavigation-`, so that it is consistent with the template name.

Comment (frenchbread) [inline docs/en.i18n.json]: Done. Good point.

**Comment by brylie** [comment]
Thanks. This can be merged once PR #44 is reviewed/merged.

**Comment by frenchbread** [comment]
@brylie Please merge.

**Pull request merged**